_includes/head.html

```diff
@@ -15,15 +15,13 @@
   <link rel="stylesheet" href="{{ '/assets/css/just-the-docs-default.css' | relative_url }}">
 
   {% if site.ga_tracking != nil %}
-    <script async src="https://www.googletagmanager.com/gtag/js?id={{ site.ga_tracking }}"></script>
-    <script>
-      window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
-
-      gtag('config', '{{ site.ga_tracking }}'{% unless site.ga_tracking_anonymize_ip == nil %}, { 'anonymize_ip': true }{% endunless %});
-    </script>
-
+    {% if site.ga_tracking.first %}
+      {% for ga_property in site.ga_tracking %}
+        {% include google_analytics.html ga_property=ga_property %}
+      {% endfor %}
+    {% else %}
+      {% include google_analytics.html ga_property=site.ga_tracking %}
+    {% endif %}
```

Comment:
If you implement the above suggestion, you won't need to include the if/else block logic here. You may also opt to bring the include back into here - that's up to you stylistically.